src/apps/testapps/testCompactCells.c

```diff
@@ -83,7 +83,9 @@ SUITE(compactCells) {
         H3Index *compressed = calloc(arrSize, sizeof(H3Index));
         t_assertSuccess(H3_EXPORT(compactCells(children, compressed, arrSize)));
         t_assert(compressed[0] == parent, "got expected parent");
-        t_assert(compressed[1] == 0, "expected only 1 cell");
+        for (int idx = 1; idx < arrSize; idx++) {
+            t_assert(compressed[idx] == 0, "expected only 1 cell");
+        }
```

Comment:
just curious why checking the second element isn't sufficient?  i'd expect the first `0` to "terminate" the array, no much how big it is.

Reply:
I'm not sure that is a valid assumption for compactCells. We should document that clearly if it is. I don't believe that assumption holds for gridDisk specifically due to the use of a hash function.

Reply:
good point! this has ramifications for bindings (at least for h3-go) where I "post-process" the internally allocated C buffer to fit the language's paradigms, e.g. I resize the returned slice of indexes to the first zero'd index for compaction...